app/models/person.rb

```diff
@@ -0,0 +1,11 @@
+class Person < ApplicationRecord
+  validates_uniqueness_of :email
+  validates :email, format: { with: URI::MailTo::EMAIL_REGEXP }
+
+  def self.email_create(email)
+    create!(email: email, created_at: Time.now)
```

Comment:
@SergeyProger doesn't `created_at` and `updated_at` get inserted/updated automatically by Rails?